### DISCUSSION TITLE
Toggle for new "Clear Selection" button

### DISCUSSION THREAD

**Discussion by loughlinclaus3** [KevinVandy/material-react-table#987] [Ideas]
Hi, can you add a toggle hidden to the table options for the new "Clear Selection" button in the `MRT_ToolbarAlertBanner`? Or alternatively, allow users to pass in props that can hide the button (without hiding the `selectedCountOfRowCountRowsSelected` localization text?

It'd be nice to be able to hide the button if I don't want users to see it, and the "select/deselect all" checkbox already performs an equivalent function when `selectAllMode` is set to `"all"`. I've got a custom theme for the header, and it kinda clashes.

If not, I could try giving it a go, and make a PR with the option. I've only been using the package for a few weeks, but I'm really liking it so far!

**Comment by KevinVandy** [comment]
That's what the `renderToolbarAlertBannerContent` is for. If you don't like the default implementation, just render your own message using the table instance apis
